test/package.json

```diff
@@ -17,7 +18,7 @@
     "bedrock-jsonld-document-loader": "^1.0.0",
     "bedrock-ledger-context": "^15.0.0",
     "bedrock-ledger-node": "digitalbazaar/bedrock-ledger-node#master",
-    "bedrock-ledger-storage-mongodb": "digitalbazaar/bedrock-ledger-storage-mongodb#op-exists",
+    "bedrock-ledger-storage-mongodb": "digitalbazaar/bedrock-ledger-storage-mongodb",
```

Comment:
@gannan08 what version of ledger storage should I use here?

Reply:
`#update-idx`

Reply:
ok cool added it here: https://github.com/veres-one/veres-one-validator/pull/35/commits/8389d10053330d5cb0c9d559c7b3d8ecc1c609cb